.github/workflows/test.yaml

```diff
@@ -42,8 +42,18 @@ jobs:
       fail-fast: true
       matrix:
         python_version: ['3.8', '3.10']
```

Comment:
Does this software not work with python 3.9? I'm curious why that's not included.

Reply:
same as above